ingest-app/src/cmr/ingest/system.clj

```diff
@@ -97,6 +98,10 @@
                                 (af/refresh-acl-cache-job "ingest-acl-cache-refresh")
                                 jvm-info/log-jvm-statistics-job))
               :caches {acl/token-imp-cache-key (acl/create-token-imp-cache)
+                       ;; Caches a map of tokens to the security identifiers
+                       context-augmenter/token-sid-cache-name (context-augmenter/create-token-sid-cache)
```

Comment:
Ok if it is assumed the SID cache is needed for user-id caching I am fine with adding it to ingest too.